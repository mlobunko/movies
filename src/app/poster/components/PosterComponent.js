import React from 'react';
import PropTypes from 'prop-types';

import {
  StyledPoster,
  StyledPosterContainer,
  StyledPosterContent
} from './styles';

export const PosterComponent = ({ posterPath }) => (
  <StyledPosterContainer>
    <StyledPosterContent>
      <StyledPoster posterPath={posterPath} />
    </StyledPosterContent>
  </StyledPosterContainer>
);

PosterComponent.displayName = 'PosterComponent';

export default PosterComponent;

PosterComponent.propTypes = {
  posterPath: PropTypes.string
};
