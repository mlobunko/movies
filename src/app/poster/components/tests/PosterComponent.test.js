import React from 'react';
import { shallow } from 'enzyme';

import { PosterComponent } from '../PosterComponent';

describe('render', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<PosterComponent posterPath={'abc'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
