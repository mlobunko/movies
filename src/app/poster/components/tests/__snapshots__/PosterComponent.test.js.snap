// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders correctly 1`] = `
<StyledPosterContainer>
  <StyledPosterContent>
    <StyledPoster
      posterPath="abc"
    />
  </StyledPosterContent>
</StyledPosterContainer>
`;
